plugins/es-repository-azure/src/test/java/org/elasticsearch/repositories/azure/AzureSnapshotIntegrationTest.java

```diff
@@ -121,4 +164,9 @@ private String httpServerUrl() {
         InetSocketAddress address = httpServer.getAddress();
         return "http://" + InetAddresses.toUriString(address.getAddress()) + ":" + address.getPort();
     }
+
+    private String invalidHttpServerUrl() {
+        InetSocketAddress address = httpServer.getAddress();
+        return "http://invalidHost:" + address.getPort();
```

Comment:
```suggestion
        return "http://dummy.invalid:" + address.getPort();
```

See https://www.rfc-editor.org/rfc/rfc2606#page-2

Reply:
thx!